### ISSUE TITLE
the same problem for some reason

### ISSUE THREAD

**Issue by Retroyawn** [CDNRae/pkhex-bulk-egg-generator#17]
I still can't see the generator in the tools menu

**Comment by CDNRae** [comment]
@Retroyawn this seems to be an issue with PKHeX and the way it loads plugins. To fix it, try going to `Options > Settings > Startup` and changing `PluginLoadMethod` to `LoadFile`. Then restart PKHeX.

**Comment by cacawate33** [comment]
Tried the fix it's not working for me either

**Comment by CDNRae** [comment]
You may have to reboot PKHeX multiple times to get the Egg Generator to appear under Tools. This issue is unfortunately out of my control, as it's related to how PKHeX loads plugins. Other plugins have this problem too, such as [Architdate's PKHeX plugins](https://github.com/architdate/PKHeX-Plugins).

**Comment by cacawate33** [comment]
Just tested opening and closing 5 times after a reboot, same problem, though using "UnsafeLoadFrom" seems to fix the issue as it showed up on the first try.

**Comment by CDNRae** [comment]
Glad it worked!

Out of curiosity, is the BulkImporter.dll file unblocked? You can check by right-clicking on it, and going to `Properties > General`. At the bottom there may be a little blurb called `Security` and to the right, a checkbox labelled `Unblock`. Unblocking it likely won't fix the load issue, but it may explain why you had to set `PluginLoadMethod` to `UnsafeLoadFrom`.

**Comment by cacawate33** [comment]
Yup! Looks like it was that way, weird but hey at least there is a workaround!
